tests/models/gptj/test_modeling_gptj.py

```diff
@@ -385,6 +388,14 @@ class GPTJModelTest(ModelTesterMixin, GenerationTesterMixin, PipelineTesterMixin
     test_model_parallel = False
     test_head_masking = False
 
+    @unittest.skipIf(not is_torch_greater_or_equal_than_1_12, reason="PR #22069 made changes that require torch v1.12+")
```

Comment:
Likely due to this line

https://github.com/njhill/transformers/blob/7888c100dadacebd4ed66adc87e24bbc5b3d6758/src/transformers/models/gptj/modeling_gptj.py#L63